docs/vgclog_examples.md

```diff
@@ -0,0 +1,594 @@
+|-------------|--------------|-------------------------------|--------------------------------------|---------------------------------------------------------------------------|
+|Scavenge     |initial        | STW              | `<gc-start>`, `<gc-end>`|Root scanning, reported as a single scavenge operation |
+|Scavenge     |intermediate        | concurrent         | `<concurrent-start>`, `<concurrent-end>`|`<warning details=>`  Root scan, live objects traversed and evacuated (copy-forwarded). Reported as a scavenge operation |
+|Scavenge     |final     | STW              | `<gc-start>`, `<gc-end>` |weak roots scanning, reported as a complex scavenge operation (`<gc-op>`) containing specific details for each of weak root groups  |
+
+### Global GC cycle
+
+The following example shows how a global GC cycle is recorded in a `gencon` policy verbose GC log. The output is broken down into sections with supporting text to explain the GC processing that is taking place.
+
+The global GC cycle runs when the *tenure* area is close to full, which typically occurs after many partial cycles. As such, the output can be found part way down the full log. For more information about the GC Initialization section, see [Verbose GC log contents and structure ](vgclog-md#verbose-gc-log-contents-and-structure). For an example log output for a `gencon` partial cycle, see [Example - `gencon`’s default partial GC cycle](./vgclogs.md/#example-gencons-default-partial-gc-cycle).
+
+ [The global GC cycle is split into 3 increments](./vgclog.md#gc-increments-and-interleaving) that interleave with partial GC cycles. Splitting the cycle operations into the following increments means that the STW pause times can be as short as possible, with the majority of the GC work being done concurrently. The interleaving can be seen in the following example, where a partial GC cycle is logged between the start and end of the global cycle.
+
+ The following elements log the GC increments and operations of the global GC cycle:
+
+ |GC Operation         | GC increment| STW or concurrent| XML element of GC increment| Details                         |
+|---------------------|-------------|-------------------------------|--------------------------------------|-----------------------|
+|n/a - initiates cycle|initial      | STW              | `<concurrent-kickoff`        |No `<gc-op>` is logged. This increment just initiates the concurrent mark increment |
+|concurrent mark      |intermediate |concurrent                     | `<gc-start>`, `<gc-end>`     |`<concurrent-trace-info>` records the progress of the concurrent mark increment|
+|final collection     |final        | STW              | `<concurrent-global-final>`  |Triggered when the card-cleaning threshold is reached. Child element is `<concurrent-trace-info reason=””>`. Operations and sub-operations include a final concurrent mark, a sweep, and an optional class unload and compact.|
+
```

Comment:
Not a must, but consider adding 'cycle-start type global' for initial increment  and 'cycle-end' for final increment in  XML elements column of the table. And then if you do add here, you should in any of tables of this if this type, too.

Reply:
Trigger reason is not always (but most often is) 'card cleaning threshold reached'. So you can say here 'such as' or 'typically'....

The other reason is 'tracing completed'. Even I don't know all details, but I believe it's if we trace objects set by targetBytes in kickoff element.

Reply:
I was under the impression that the <cycle-start> and <cycle-end> elements do not mark the beginning or end of increments, that it was the <gc-start>, <concurrent-kickoff> etc. elements that mark the beginning of increments....? This definition  also fits well with the definitions we are using for partial GC and the Balanced cycles.

Reply:
Change to trigger reason: thanks, actioned: 
"The increment is typically triggered when a card cleaning threshold is reached. The completion of a tracing phase can also trigger the increment."